pkg/appgw/ingress_rules_test.go

```diff
@@ -129,14 +129,14 @@ var _ = Describe("Process ingress rules, listeners, and ports", func() {
 			Expect(len(frontendListeners)).To(Equal(1))
 		})
 		It("should have correct number of front end ports", func() {
-			Expect(len(frontendPorts.ToSlice())).To(Equal(1))
+			Expect(len(frontendPorts)).To(Equal(1))
 		})
 		It("should have a listener on port 443", func() {
 			actualListener := getMapKeys(&frontendListeners)[0]
 			Expect(actualListener.FrontendPort).To(Equal(port443))
 		})
 		It("should have one port 443", func() {
-			actualPort := frontendPorts.ToSlice()[0]
+			actualPort := getInt32MapKeys(&frontendPorts)[0]
 			Expect(actualPort).To(Equal(port443))
```

Comment:
we can use `ContainElement` here ?

Reply:
Yes - now that we have Ginkgo's luxury of `ContainElement` - we should use it everywhere!